2018-02-16T13:23:52.535Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'Participant:org.kibaati.Person' in collection with ID '$sysregistries' does not exist)
    at /usr/local/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-16T13:23:52.539Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Participant:org.kibaati.Person' in collection with ID '$sysregistries' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'Participant:org.kibaati.Person' in collection with ID '$sysregistries' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/usr/local/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)"}$
